graphql_compiler/schema_generation/sqlalchemy/edge_descriptors.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import namedtuple
+
+SQLEdgeDescriptor = namedtuple(
```

Comment:
I'd call this a `ForeignKeyEdgeDescriptor` or `DirectEdgeDescriptor`. Now you're implying the `JunctionTableEdgeDescriptor` is not a sql edge descriptor.

Reply:
Ok.